File: WebUI/UI/src/app/datas/models/details/multiple-choice-detail.ts
export class MultipleChoiceDetail {
  constructor(
    multipleChoiceQuestionId: number,
    choiceOrder: number,
    title: string,
    isOtherOption: boolean
  ) {
    this.multipleChoiceQuestionId = multipleChoiceQuestionId;
    this.choiceOrder = choiceOrder;
    this.title = title;
    this.isOtherOption = isOtherOption;
  }

  multipleChoiceDetailId: number;
  multipleChoiceQuestionId: number;
  choiceOrder: number;
  title: string;
  isOtherOption: boolean;
}

File: Entities/Details/MultipleChoiceDetail.cs
using System.ComponentModel.DataAnnotations;
using Core.Entities;

namespace Entities.Details
{
    public class MultipleChoiceDetail : IEntity
    {
        //public MultipleChoiceDetail()
        //{
        //    Letters = new List<Letter>();
        //}

        [Key]
        public int MultipleChoiceDetailId { get; set; }
        public int MultipleChoiceQuestionId { get; set; }
        public int ChoiceOrder { get; set; }
        public string Title { get; set; }
        public bool isOtherOption { get; set; }
       // public MultipleChoiceQuestion MultipleChoiceQuestion { get; set; }
       // public List<Letter> Letters { get; set; }
    }
}
